src/openst.js

```diff
@@ -68,9 +70,60 @@ class OpenST {
       createAndAddModules: createAndAddModules.address,
       tokenRules: tokenRulesAddress,
     };
+
     Object.assign(this.chainConfig.openst, setupData);
+
+    this.chainConfig.utilityBrandedTokens[ubtConfig.index].openst = setupData;
+
     logger.info('Completed Setup of OpenST');
   }
+
+  getUtilityBrandedTokenConfig(auxiliaryEIP20Token) {
+    for (let i = 0; i < this.chainConfig.utilityBrandedTokens.length; i++) {
+      const ut = this.chainConfig.utilityBrandedTokens[i];
+      if (ut.address === auxiliaryEIP20Token) {
+        return { ubt: ut, index: i };
+      }
+    }
+    return 0;
+  }
```

Comment:
Above method can be simplified like below:
```
getUtilityBrandedTokenConfig(auxiliaryEIP20Token) {
    return this.chainConfig.utilityBrandedTokens.find(
      ut => ut. address === auxiliaryEIP20Token,
    );
  }
```

Reply:
Above approach will only return the config of UBT. I also need the index in the list of `utilityBrandedTokens` to update it for pricer rule address.

Reply:
ok
We can keep the returns structure consistent.